src/routes/(admin)/questionbank/single-question/[id]/+page.server.ts

```
@@ -0,0 +1,72 @@
+import type { PageServerLoad, Actions } from './$types.js';
+import { questionSchema } from './schema.js';
+import { superValidate } from 'sveltekit-superforms';
+import { zod } from 'sveltekit-superforms/adapters';
+import { getSessionTokenCookie } from '$lib/server/auth.js';
+import { fail, redirect } from '@sveltejs/kit';
+import { BACKEND_URL } from '$env/static/private';
+
+
+export const load: PageServerLoad = async ({ params }: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)